dev/Breadcrumb/Breadcrumb.cpp

```diff
@@ -0,0 +1,561 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+#include "common.h"
+#include "Vector.h"
+#include "Breadcrumb.h"
+#include "RuntimeProfiler.h"
+#include "ResourceAccessor.h"
+
+#include "BreadcrumbItem.h"
+#include "BreadcrumbLayout.h"
+#include "BreadcrumbItemClickedEventArgs.h"
+
+Breadcrumb::Breadcrumb()
+{
+    __RP_Marker_ClassById(RuntimeProfiler::ProfId_Breadcrumb);
+
+    SetDefaultStyleKey(this);
+    m_breadcrumbElementFactory = winrt::make_self<BreadcrumbElementFactory>();
+}
+
+void Breadcrumb::OnApplyTemplate()
+{
+    winrt::IControlProtected controlProtected{ *this };
+
```

Comment:
All revokers need to be revoked here. See PagerControl::OnApplyTemplate() for instance.

m_item**s**RepeaterElementPreparedRevoker.Revoke();
m_item**s**RepeaterElementIndexChangedRevoker.Revoke();
m_item**s**RepeaterElementClearingRevoker.Revoke();
m_breadcrumbItem**s**RepeaterLoadedRevoker.Revoke();